paperqa/core.py

```diff
@@ -20,10 +20,14 @@ def llm_parse_json(text: str) -> dict:
     def escape_newlines(match: re.Match) -> str:
         return match.group(0).replace("\n", "\\n")
 
+    def escape_double_backslashes(match: re.Match) -> str:
+        return match.group(0).replace("\\", "\\\\")
```

Comment:
Do you mind doing this in a separate PR, with a unit test addition?

Reply:
https://github.com/Future-House/paper-qa/pull/858